nx/source/services/set.c

```diff
@@ -5,6 +5,8 @@
  * @author yellows8
  * @copyright libnx Authors
  */
+#include <string.h>
+
```

Comment:
remove newline

Reply:
I was just following the way it's used in acc.c -> https://github.com/switchbrew/libnx/blob/master/nx/source/services/acc.c#L1 but okay.

Reply:
Heh, no newline elsewhere :|

Reply:
Er nvm revert this, original didn't have newline & nothing(?) else has newline here either.